docs/detections/prebuilt-rules-management.asciidoc

```diff
@@ -108,18 +107,85 @@ NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules
++
 
-. Do one of the following to update prebuilt rules on the *Rules* page:
+. From the *Rule Updates* tab, do one of the following to update prebuilt rules:
 * Update all available rules: Click *Update all*.
-* Update a single rule: Click *Update rule* for that rule.
+* Update a single rule: Click *Update rule* for that rule. 
 * Update multiple rules: Select the rules and click *Update _x_ selected rule(s)*.
 +
 TIP: Use the search bar and *Tags* filter to find the rules you want to update. For example, filter by `OS: Windows` if your environment only includes Windows endpoints. For more on tag categories, refer to <<prebuilt-rule-tags>>.
+
+[float]
+[[rule-update-statuses]]
+==== Rule update statuses
+
+This table describes statuses that might appear for rule fields being updated.  
+
+[cols="2"]
+|===
+
+| *Ready for update*
+a| Displays when there are no conflicts to resolve. 
+
+Further action is not required for the field. It is ready to be updated.
+
+| *No update*
+a| Displays when the field is not being updated by Elastic, but the current field value differs from the original one. This typically happens when the field's value was changed after the prebuilt rule was initially installed.
+
+Further action is not required for the field. It is ready to be updated.
+
+TIP: You can still change the final field update, if needed. To do so, make your changes in the *Final update* section and save them.
+
+| *Review required*
+a| Displays when Elastic auto-resolves a conflict between the current field value and the value from the incoming Elastic update. 
+
+You must accept or edit the field's final update and save the changes. Refer to <<resolve-reduce-rule-conflicts>> to learn more about auto-resolved conflicts and how to reduce future conflicts.
+
+| *Action required*
+a| Displays when Elastic could not auto-resolve the conflict between the current field value and the value from the incoming Elastic update. 
+
+You must manually set and save the field's final update. Refer to <<resolve-reduce-rule-conflicts>> to learn more about conflicts that need manual fixes and how to reduce future conflicts.
+
+|===
+
+
+[float]
+[[resolve-reduce-rule-conflicts]]
+==== Resolve and reduce update conflicts
+
+Keeping prebuilt rule updated might help you minimize the frequency and complexity of conflicts that occur during rule updates.
+
+When a conflict does happen, Elastic attempts to resolve it and will suggest a fix for your review. In these cases, you can accept or edit the suggested fix, then update the rule. If Elastic can't resolve the conflict, you must manually fix it before updating the rule. 
+
+To manually fix a conflict:
+
+. Find the rule field with the unresolved conflict. 
++
+TIP: Fields with unresolved conflicts will have the `Action required` badge next to their names.
++
+. Go to the *Final update* section and do any of the following:
+** Keep your changes and reject the Elastic update.
```

Comment:
I don't think we have an action to reject the Elastic update. Does it mean that user keeps the suggested field (could be current version or Elastic) vs Editing?

Reply:
Reworded in [b5b5fc4](https://github.com/elastic/security-docs/pull/6568/commits/b5b5fc41d6ddeeaad0dfa58ea62a85d6411920cd)